### PR TITLE
Update CLI to support Visibility Archival

### PR DESCRIPTION
**Pull request by yycptt**
https://github.com/uber/cadence/issues/2496

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,12 +5,10 @@ go 1.12
 require (
 	github.com/DataDog/zstd v1.4.0 // indirect
 	github.com/Shopify/sarama v1.23.0
-	github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239 // indirect
```
Comment (andrewjdawson2016) [inline go.mod]: I upgraded to go 1.13 and I did not get any changes in these files can you try upgrading to that?

File: tools/cli/flags.go [hunk 2]
```diff
@@ -447,6 +447,48 @@ func getFlagsForScan() []cli.Flag {
...
+func getFlagsForListArchived() []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name:  FlagListQueryWithAlias,
+			Usage: "SQL like query. Please check the documentation of the visibility archiver used by your domain for detailed instructions",
+		},
+		cli.IntFlag{
+			Name:  FlagPageSizeWithAlias,
+			Value: 100,
+			Usage: "Result page size",
```
Comment (andrewjdawson2016) [inline tools/cli/flags.go]: "Count of visibility records included in a single page"

File: tools/cli/flags.go [hunk 3]
```diff
@@ -447,6 +447,48 @@ func getFlagsForScan() []cli.Flag {
...
+		},
+		cli.BoolFlag{
+			Name:  FlagPrintRawTimeWithAlias,
+			Usage: "Print raw time stamp",
```
Comment (andrewjdawson2016) [inline tools/cli/flags.go]: "timestamp"

File: tools/cli/flags.go [hunk 4]
```diff
@@ -447,6 +447,48 @@ func getFlagsForScan() []cli.Flag {
 	}
 }
 
+func getFlagsForListArchived() []cli.Flag {
```
Comment (vancexu) [inline tools/cli/flags.go]: Let extract some common flags from list commands and reuse. This will help people extend common feature.

Comment (yycptt) [reply tools/cli/flags.go]: Done.

**Pull request merged**